src/lib.rs

```diff
@@ -139,71 +151,50 @@ fn deserialize_frames<'de, D: serde::Deserializer<'de>>(de: D) -> Result<Vec<Fra
 }
 
 #[derive(Serialize, Deserialize, Debug, PartialEq, Eq, Clone, Copy)]
+#[serde(rename_all = "camelCase")]
+#[non_exhaustive]
 pub enum Direction {
-    #[serde(rename="forward")]
     Forward,
-    #[serde(rename="reverse")]
     Reverse,
-    #[serde(rename="pingpong")]
     Pingpong,
 }
 
 #[derive(Serialize, Deserialize, Debug, PartialEq, Eq, Clone)]
+#[serde(rename_all = "camelCase")]
+#[non_exhaustive]
 pub struct Frametag {
     pub name: String,
     pub from: u32,
     pub to: u32,
     pub direction: Direction,
-
-    #[doc(hidden)] #[serde(skip)] pub _non_exhaustive: (), // more fields may be added (color? data?)
 }
 
 // These are listed at:
 // https://github.com/aseprite/aseprite/blob/51b038ac024dd99902ab5b0c0d61524c48856b93/src/doc/blend_mode.cpp#L18-L37
 
 #[derive(Serialize, Deserialize, Debug, PartialEq, Eq, Clone, Copy)]
+#[serde(rename_all = "camelCase")]
+#[non_exhaustive]
 pub enum BlendMode {
-    #[serde(rename="normal")]
     Normal,
-    #[serde(rename="multiply")]
     Multiply,
-    #[serde(rename="screen")]
     Screen,
-    #[serde(rename="overlay")]
     Overlay,
-    #[serde(rename="darken")]
     Darken,
-    #[serde(rename="lighten")]
     Lighten,
-    #[serde(rename="color_dodge")]
```

Comment:
Weird. Here the variants were renamed into snake_case. I don’t know why from merely looking at it now though. Do you know, by chance?

Why did you change it to camelCase? Out of conformity to the other cases?